.github/workflows/build-xmtpd.yml

```
@@ -43,13 +43,20 @@ jobs:
             type=ref,event=tag
             type=ref,event=pr
             type=sha
-
+      - name: Set up Docker image file based on the matrix variable
+        id: set_dockerfile
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:33: Double quote to prevent globbing and word splitting

(shellcheck)